tysnj/fitlit
# Feature/activity display

## Is this a fix or a feature?
Feature

## What is the change?
Completes the charts and data visualization for all of the activity metrics. Also adds all-time sleep data to the bottom. 

## What does it fix?
Updates the step chart to show accurate user and all user step data for specific day.

## Where should the reviewer start?
Fetch the branch feature/activity-display

## How should this be tested?
Check the top 6 icons, stacked bar charts, and line charts for accuracy/display. Check that the sleep totals are displayed in the circles towards the bottom of the screen.
